src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -30,15 +30,23 @@ const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
   const key = _.isString(authenticatorKey) ? authenticatorKey.toLowerCase() : '';
   let authenticatorData = {};
   switch (key) {
-  case AUTHENTICATOR_KEY.EMAIL:
+  case AUTHENTICATOR_KEY.EMAIL: {
+    const emailType = authenticator.relatesTo;
     Object.assign(authenticatorData, {
       description: isVerifyAuthenticator
         ? ''
         : loc('oie.email.authenticator.description', 'login'),
       iconClassName: 'mfa-okta-email',
       buttonDataSeAttr: getButtonDataSeAttr(authenticator),
     });
-    break;
+
+    //TODO: backend should send a new key so that this can be handled gracefully like other authenticators
+    if(emailType && emailType.displayName === 'Secondary Email') {
```

Comment:
once the auth key is changed this can be removed before we merge.. https://github.com/okta/okta-signin-widget/pull/2122/files#r679367383